scripts/transfer.js

```diff
@@ -1,16 +1,27 @@
-// Transfer 10,000 sarco tokens to the an address
-// transferAddress defaults to the embalmer address the webapp uses
+// Transfer 10,000 sarco tokens to an address
+// transferAddress defaults to the first unnamed hardhat local node account
 const amountToTransfer = "10000000000000000000000";
-const transferAddress =
-  process.env.TRANSFER_ADDRESS || "0x5FbDB2315678afecb367f032d93F642f64180aa3";
+const mockTokenAddress = "0x5fbdb2315678afecb367f032d93f642f64180aa3";
```

Comment:
`0x5fbdb2315678afecb367f032d93f642f64180aa3` is actually the mock sarco token address on localhost. Didn't notice until I tried to run this script myself

Reply:
@DarksightKellar -- good catch -- note to self -- run the code after making changes.